src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -415,16 +457,19 @@ private void validateAsset(AssetModel asset, Errors errors, SchemaValidationCont
   }
 
   private void validateSchema(DatasetSpecificationModel schema, Errors errors) {
-    SchemaValidationContext context = new SchemaValidationContext();
+    SchemaValidationContext context =
+        new SchemaValidationContext(SchemaValidationContext.Operation.CREATE);
     List<TableModel> tables = schema.getTables();
     if (tables.isEmpty()) {
       errors.rejectValue(
-          "schema", "IncompleteSchemaDefinition", "Dataset tables must be defined in the schema");
+          context.getFieldName(),
```

Comment:
I could change back the "schema"s in this method because the context is made with CREATE operation here, so they will always be "schema"

Reply:
👍 Assuming we're setting the field name in both cases, I think it makes sense to move away from the hard coded value where ever possible.